### ISSUE TITLE
V1 - Footer is different from landing page in CLA Consoles

### ISSUE THREAD

**Issue by nirupamav** [communitybridge/easycla#2081]
**Footer in landing page**
![image.png](https://images.zenhubusercontent.com/5eda192d0167eac230ac5cee/5f96fb4d-67a5-4422-a31e-d2a650f53515)

**Footer in Project Console, Corporate Console and Contributor Console**
![image.png](https://images.zenhubusercontent.com/5eda192d0167eac230ac5cee/4335514c-f180-46ea-8453-6863e870934f)

**Comment by amolsontakke3576** [comment]
@nirupamav  It is as designed as per suggestion on ticket #1813 by @dealako 

Please follow this link: https://app.zenhub.com/workspaces/easyclav2-5ed619832c71a855a76ef404/issues/communitybridge/easycla/1813

**Comment by nirupamav** [comment]
@amolsontakke3576 
Yes. As per that, even in cla consoles we should have same footer right?

**Comment by dealako** [comment]
I have raised a question in #pmm to ask about a common footer.

**Comment by dealako** [comment]
@nirupamav we can close this ticket once #2097 is resolved
